### PR TITLE
NaHCO3bc-finalproject

### PR DESCRIPTION
**Pull request by NaHCO3bc**
Here is my finalproject

### CODE REVIEW

**Comment by NaHCO3bc** [comment]
Upadte my README.md

**Comment by NaHCO3bc** [comment]
Finishing the third homework by updaing and amending the txt.md

**Comment by NaHCO3bc** [comment]
i need a reviewer @MuZhou233

**Comment by MuZhou233** [comment]
Fix the conflicts and pass the check action first

**Comment by NaHCO3bc** [comment]
I have resolved the conflict. @MuZhou233

File: _2021-finalproject/README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ PR 考察分两阶段
 2. 第二阶段要求在11月10日21:00前完成合并
    - 值得注意的是，在第二阶段仍然可以修改或增加你的作业内容，但建议你尽量在这之前完成
    - 我们会在两个阶段的临界点修改其中一个检查项目，你需要在这之后再次通过所有检查项目，通过后再次 request review，如果没有问题，你的 PR 就会被合并
+   - 被修改的检查项目为：作业提交文件夹变更为 `_2021-finalproject`
```
Comment (MuZhou233) [inline _2021-finalproject/README.md:24]: Don't change this file in your commits

**Review by MuZhou233 — CHANGES_REQUESTED** [review]

File: _2021-finalproject/README.md (hunk 1 above)
Comment (NaHCO3bc) [reply _2021-finalproject/README.md:24]: I delete the README.md at the beginning,so i have to touch a new README.md in my terminal.Can i use git reset --hard?

Comment (NaHCO3bc) [reply _2021-finalproject/README.md:24]: @MuZhou233

Comment (MuZhou233) [reply _2021-finalproject/README.md:24]: give it a try

Comment (NaHCO3bc) [reply _2021-finalproject/README.md:24]: Do i solve the problem?

Comment (MuZhou233) [reply _2021-finalproject/README.md:24]: No

**Pull request closed without merging**